jetty-documentation/src/main/asciidoc/configuring/jsp/configuring-jsp.adoc

```diff
@@ -179,13 +179,6 @@ By default, the Glassfish JSP container tries to use the compiler that is part o
 If you do not have a full JDK, you can configure the Eclipse Java Compiler that Jetty ships in the distro in `{$jetty.home}/lib/jsp/.`
 You need to define a SystemProperty that prevents the Glassfish JSP engine from defaulting to the in-JVM compiler.
```

Comment:
We should remove all discussion of Glassfish JSP going forward, as that is only valid for older versions of jetty.